content/blog/2022-07-FGBS1/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Fighting Galactic Bullshit 1"
+tease: "Dealing with misconceptions one blog post at a time"
+authors: "The Galaxy Team"
+date: "2022-07-11"
+---
+
+![galaxy hops, super smooth](hops.jpg)
+
+Galaxy is a free open platform designed to give researchers across the world unconstrained access to tools and infrastructure necessary for analysis of their data. Galaxy does a lot of things and contains many components, making it difficult to quickly comprehend everything that it can do. It also evolves [rapidly](https://docs.galaxyproject.org/en/master/releases/index.html). As a result, there are a number of misconceptions about what it is and what can (or cannot) be done with it. To spread the ultimate knowledge across the Universe, we are starting the **“Fighting Galactic Bullshit (FGBs)”** series (yes, we were definitely inspired by a certain [effort](https://www.callingbullshit.org/)). This series identifies the most common misconceptions and explains why they are, simply put, BULLSHIT! It will include the following “episodes”:
+
+1. Galaxy is only useful for genome scientists…, **But, all scientists have genomes, right?**
```

Comment:
I guess that's the point of the blog post to come, but seems very cryptic. Not sure we'll be shooting ourselves in the foot with genome and dbkey plastered all over the interface.

Reply:
It's about __only useful__ for genome scientists. The first blog post will cover this topic and will point to other communities using Galaxy.

Reply:
Sure, but what are we alluding to with `But, all scientists have genomes, right` ?

Reply:
I see your confusion. But this refers to "the plant next to me has a genome" :)
If that is too confusing we might need to find a new word here.

Reply:
suggestion: 
What is a genome scientist anyway?

Reply:
I'm saying this is cryptic. Is the joke that because everyone has a genome everyone is a genome scientist (your comp-sci employees, all philosophers, and Object Orientation would like a word with you 🤣 ) ? Or is it that it's irrelevant that Galaxy started that way ?